swagger.yaml

```diff
@@ -6614,6 +6614,7 @@ definitions:
       - 'lvm-requirements-satisfied'
       - 'mce-requirements-satisfied'
       - 'network-type-valid'
+      - 'oci-manifests-satisfied'
```

Comment:
I think we should add something more generic for other platforms that we will have in the future, something like: invalid_platform_configuration and the description will be more specific about OCI
@avishayt WDYT?

Reply:
I agree since we'll have at most one platform selected at a time.  "platform-requirements-satisfied" seems to fit with the others.

Reply:
"configuration" seems appropriate

Reply:
> "configuration" seems appropriate

please talk with UI about this new category before implementing it - they need to decide where to display this validation and maybe they are using the category
@jkilzi @nirfarkas FYI

Reply:
The category already exists.  It's currently only used for "pull secret set".